app/views/about/more.json.rabl

```diff
@@ -0,0 +1,10 @@
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
```

Comment:
I was just going from a default elsewhere. Happy to change this if there's a faster way to do it.